.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build package
+
+on:
+  pull_request:
+    branches:
+     - trunk
+
+jobs:
+  Build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [ '3.7', '3.8', '3.9' ]
+    name: Build test on ${{ matrix.os }} with pytest ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install requirements
+        run: |
+          pip install -r requirements.txt
```

Comment:
Do we need to install the requirements? I thought we were testing that the pip install would do this?